pkgs/applications/display-managers/sddm/default.nix

```diff
@@ -93,6 +88,4 @@ stdenv.mkDerivation {
 
     runHook postInstall
   '';
-
-  inherit (unwrapped) meta;
```

Comment:
No, in Nix `{ inherit (foo) bar; }` desugars to `{ bar = foo.bar; }` and `{ inherit baz; }` to `{ baz = baz; }`. So with `inherit unwrapped;` this package would have an `unwrapped` attribute but not a `meta` attribute.